packages/team-management/src/components/TeamsView/index.tsx

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Row, Col, Button, Input, notification } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+import TeamsDetail, { TeamsDetailProps } from '../TeamsDetail';
+import { SearchOutlined } from '@ant-design/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { OpenSRPService } from '@opensrp/server-service';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import reducer, {
+  fetchOrganizationsAction,
+  getOrganizationsArray,
+  Organization,
+  reducerName,
+} from '../../ducks/organizations';
+import { getAccessToken } from '@onaio/session-reducer';
+import { API_BASE_URL, TEAMS_ALL, TEAM_PRACTITIONERS, URL_ADD_TEAM } from '../../constants';
+import Table, { TableData } from './Table';
+import './TeamsView.css';
+import { Spin } from 'antd';
+import { Link } from 'react-router-dom';
+import { Practitioner } from '../../ducks/practitioners';
+
+reducerRegistry.register(reducerName, reducer);
+
+export const loadSingleTeam = (
+  row: TableData,
+  accessToken: string,
+  setDetail: (isLoading: string | Organization) => void,
+  setPractitionersList: (isLoading: string | Practitioner[]) => void
+): void => {
+  const serve = new OpenSRPService(accessToken, API_BASE_URL, TEAM_PRACTITIONERS + row.identifier);
+  serve
+    .list()
+    .then((response: Practitioner[]) => {
+      setPractitionersList(response);
+      setDetail(row);
+    })
+    .catch((e) => notification.error({ message: `${e}`, description: '' }));
+};
+
+const TeamsView: React.FC = () => {
+  const dispatch = useDispatch();
+  const accessToken = useSelector((state) => getAccessToken(state) as string);
+  const teamsArray = useSelector((state) => getOrganizationsArray(state));
+  const [detail, setDetail] = useState<TeamsDetailProps | null>(null);
+  const [practitionersList, setPractitionersList] = useState<Practitioner[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [value, setValue] = useState('');
+  const [filter, setfilterData] = useState<TableData[] | null>(null);
+
+  useEffect(() => {
+    if (isLoading) {
+      const serve = new OpenSRPService(accessToken, API_BASE_URL, TEAMS_ALL);
+      serve
+        .list()
+        .then((response: Organization[]) => {
+          dispatch(fetchOrganizationsAction(response));
+          setIsLoading(false);
+        })
+        .catch((e) => notification.error({ message: `${e}`, description: '' }));
```

Comment:
Let's use the @opensrp/notifications package. Also, let's display to the use the generic error message **An error occurred**